### PR TITLE
fixes 1736. ensure check-i18n command outputs a utf-8 file to support…

### PR DESCRIPTION
**Pull request by thecapdan**
… non ascii languages

### CODE REVIEW

File: plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java [hunk 1]
```diff
@@ -143,17 +146,35 @@ private void generateAllTranslationsCSV(String appName) {
...
+			}
+			catch (UnsupportedEncodingException e) 
+			{
+				System.out.println(e.getMessage());
+			}
+			catch (IOException e)
+			{
+				System.out.println(e.getMessage());
+			}
+			catch (Exception e)
```
Comment (briandipalma) [inline plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java]: `catch (Exception e)` is that good practice? Seems odd but I'm not a Java dev.

File: brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js [hunk 2]
```diff
@@ -256,7 +256,7 @@
 			throw new Error("No definition for module " + id + " has been loaded.");
 		}
 
-		// For closer spec compliance we should define id as a nonconfigurable, nonwritable
+		// For closer spec compliance, we should define id as a nonconfigurable, nonwritable
 		// property, but this at least works OK in non-es5 browsers (like ie8).
```
Comment (briandipalma) [inline brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js]: Adding the `,` as a workaround for the weird Chrome bug?

File: plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java (hunk 1 above)
Comment (thecapdan) [inline plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java]: yup. I'll address that when i fix the spec test which is failing, there must be a test that expects garbled encoding:b

File: plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java [hunk 3]
```diff
@@ -143,17 +146,35 @@ private void generateAllTranslationsCSV(String appName) {
...
 				writer.append(StringUtils.join(row, ",")+"\n");
-			}			
-		} catch (IOException e) {
-			e.printStackTrace();
+			}
+			writer.flush();
+			writer.close();
+
+			String content = FileUtils.readFileToString(file, "UTF-8");
+	        FileUtils.write(file, content, "UTF-8");
```
Comment (briandipalma) [inline plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java]: Weird indentation.

File: brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js (hunk 2 above)
Comment (thecapdan) [inline brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js]: yup.  it was either that or having to write a preposterous release note

File: plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java (hunk 3 above)
Comment (thecapdan) [inline plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java]: ty. will fix

**Comment by briandipalma** [comment]
Only minor nits/questions.

File: plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java (hunk 1 above)
Comment (briandipalma) [inline plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java]: 👍

**Pull request merged**